src/IconButton/InteractiveIcon.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import useActiveElementKeypress from '../hooks/useActiveKeyPress';
+import newId from '../utils/newId';
+
+const iconPrefix = 'interactive-icon';
+
+const InteractiveIcon = React.memo(({
```

Comment:
Why use `React.memo` here?

Reply:
Oh, I was having a weird thing happen where it was re-rendering too often, and this was one of the suggestions I found for fixing it. But I'm pretty sure something else fixed it in the end; I'll remove.